### ISSUE TITLE
Cannot grant any group group-model permissions

### ISSUE THREAD

**Issue by ironfroggy** [ansible/galaxy_ng#455]
##### SUMMARY
On the group edit page I can add all permissions to a group as an admin user, but I cannot add the permissions to work with Groups themselves.

##### STEPS TO REPRODUCE

- As an admin user create a group
- Navigate to edit the group
- Select any group permissions
- Try to save

##### EXPECTED RESULTS

The group should be granted the permissions given in the form.

##### ACTUAL RESULTS

Errors appear for each selected permission. The permissions are not saved for the group. All other permissions _are_ saved.
![cannot-edit-group-perms](https://user-images.githubusercontent.com/34982/93261459-b4cdb780-f770-11ea-8715-40d81be37d38.jpg)

**Comment by ironfroggy** [comment]
```api_1               | pulp: django.request:ERROR: Internal Server Error: /api/automation-hub/_ui/v1/groups/1/model-permissions/
api_1               | Traceback (most recent call last):
api_1               |   File "/venv/lib64/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
api_1               |     response = get_response(request)
api_1               |   File "/venv/lib64/python3.6/site-packages/django/core/handlers/base.py", line 115, in _get_response
api_1               |     response = self.process_exception_by_middleware(e, request)
api_1               |   File "/venv/lib64/python3.6/site-packages/django/core/handlers/base.py", line 113, in _get_response
api_1               |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
api_1               |   File "/venv/lib64/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
api_1               |     return view_func(*args, **kwargs)
api_1               |   File "/venv/lib64/python3.6/site-packages/rest_framework/viewsets.py", line 114, in view
api_1               |     return self.dispatch(request, *args, **kwargs)
api_1               |   File "/venv/lib64/python3.6/site-packages/rest_framework/views.py", line 505, in dispatch
api_1               |     response = self.handle_exception(exc)
api_1               |   File "/venv/lib64/python3.6/site-packages/rest_framework/views.py", line 465, in handle_exception
api_1               |     self.raise_uncaught_exception(exc)
api_1               |   File "/venv/lib64/python3.6/site-packages/rest_framework/views.py", line 476, in raise_uncaught_exception
api_1               |     raise exc
api_1               |   File "/venv/lib64/python3.6/site-packages/rest_framework/views.py", line 502, in dispatch
api_1               |     response = handler(request, *args, **kwargs)
api_1               |   File "/venv/lib64/python3.6/site-packages/pulpcore/app/viewsets/user.py", line 133, in create
api_1               |     permission = self.get_model_permission(request)
api_1               |   File "/venv/lib64/python3.6/site-packages/pulpcore/app/viewsets/user.py", line 101, in get_model_permission
api_1               |     return get_object_or_404(Permission, **data)
api_1               |   File "/venv/lib64/python3.6/site-packages/django/shortcuts.py", line 93, in get_object_or_404
api_1               |     return queryset.get(*args, **kwargs)
api_1               |   File "/venv/lib64/python3.6/site-packages/django/db/models/query.py", line 412, in get
api_1               |     (self.model._meta.object_name, num)
api_1               | django.contrib.auth.models.Permission.MultipleObjectsReturned: get() returned more than one Permission -- it returned 2!
```

**Comment by chouseknecht** [comment]
Duplicate of #458

**Comment by chouseknecht** [comment]
For now we're going to comment out the `groups` field. Will put it back and fix in the `4.2.0rc` release.
